### ISSUE TITLE
[FEATURE] Add means of selecting sounds within a soundfont via the mixer

### ISSUE THREAD

**Issue by bkunda** [musescore/MuseScore#16809]
This is a priority task for 4.1, and would resolve https://github.com/musescore/MuseScore/issues/15190. 

This issue description will be replaced with a design specification in due course.

**Comment by Tantacrul** [comment]
@jessjwilliamson - this is ready right? Can you attach the Figma link for it here?

Also - if the search box aspect of the design is ready - can you pop that in the 'Community' column?

**Comment by Tantacrul** [comment]
Also, once you've confirmed this design is ready, can you un-assign yourself and then tell the developers?

@bkunda - for visibility

**Comment by jessjwilliamson** [comment]
Choosing Soundfonts task here! https://github.com/musescore/MuseScore/issues/17233

**Comment by Tantacrul** [comment]
Ah, you created a new task.

I think it makes more sense to add designs to this original tickets (this one) - or to close the original ticket when you create a new task.

Anyway, I'll close this now, so we don't have duplicates. Cheers!

**Comment by Tantacrul** [comment]
Duplicate issue.

**Comment by jessjwilliamson** [comment]
My bad! I'll check next time :)
